### ISSUE TITLE
Tutorials that need improvement

### ISSUE THREAD

**Issue by indhub** [apache/incubator-mxnet#6493]
Here is the list of tutorials that were removed from the website for not meeting the quality bar. These tutorials should be improved, reviewed and committed.

- [How do I work with MXNet within a Matlab environment?](https://github.com/dmlc/mxnet/tree/master/matlab)
- [Can I run MXNet in a browser using JavaScript?](https://github.com/dmlc/mxnet.js/)
- [Object Detection using Faster R-CNN](https://github.com/dmlc/mxnet/tree/master/example/rcnn)
- [Object Detection using SSD](https://github.com/dmlc/mxnet/tree/master/example/ssd)
- [Neural Art - Transfer the style of one image onto the content the content of another image](https://github.com/dmlc/mxnet/tree/master/example/neural-style)
- [Large Scale Image Classification](https://github.com/dmlc/mxnet/tree/master/example/image-classification)

### Natural Language Processing
- [Text Classification using Convolutional Neural Networks](http://mxnet.io/tutorials/nlp/cnn.html)
- [NCE Loss - Speed up text classification with large output layers](http://mxnet.io/tutorials/nlp/nce_loss.html)

### Speech Recognition
- [Phoneme Classification - Use LSTM recurrent nets to recognize phonemes in audio](http://mxnet.io/tutorials/speech_recognition/speech_lstm.html)
- [Baidu Warp CTC - Jointly learn predictions and alignments with CTC loss](http://mxnet.io/tutorials/speech_recognition/baidu_warp_ctc.html)

### Unsupervised Learning and Generative Modeling

- [Generative Adversarial Networks](http://mxnet.io/tutorials/unsupervised_learning/gan.html)

- [Autoencoders - Find low dimensional representations of data](http://mxnet.io/tutorials/unsupervised_learning/auto_encoders.html)

- [Matrix Factorization - Discover latent factors of user preference in MovieLens data](http://mxnet.io/tutorials/python/matrix_factorization.html)

- [Recommender Systems - Build a complete recommender system with matrix factorization](http://mxnet.io/tutorials/general_ml/recommendation_systems.html)

### R

- [Neural Networks with MXNet in Five Minutes](http://mxnet.io/tutorials/r/fiveMinutesNeuralNetwork.html)

- [Classifying Handwritten Digits with Convolutional Neural Networks](http://mxnet.io/tutorials/r/mnistCompetition.html)

- [Classify Real-world Images with a Pre-trained Model](http://mxnet.io/tutorials/r/classifyRealImageWithPretrainedModel.html)	

- [Dogs vs. Cats Classification with Fine-tuning](https://statist-bhfz.github.io/cats_dogs_finetune)

- [Character-Level Language Modeling with LSTM RNNs](http://mxnet.io/tutorials/r/charRnnModel.html)

### Scala

- [Create MXNet Scala Applications with the IntelliJ IDE](http://mxnet.io/tutorials/scala/mxnet_scala_on_intellij.html)

- [Handwritten Digit Classification with Multilayer Perceptrons](http://mxnet.io/tutorials/scala/mnist.html)

- [Character-Level Language Modeling with LSTM RNNs](http://mxnet.io/tutorials/scala/char_lstm.html)

### C++

- [Basics](http://mxnet.io/tutorials/c++/basics.html)		

### Perl		

- [Calculator, handwritten digits and roboshakespreare](http://blogs.perl.org/users/sergey_kolychev/2017/04/machine-learning-in-perl-part2-a-calculator-handwritten-digits-and-roboshakespeare.html)

**Comment by thirdwing** [comment]
@indhub Can you give me more information on the **quality bar**?

Thank you!

**Comment by piiswrong** [comment]
I suggest we start with Faster-RCNN and SSD. They are quite important

**Comment by hadjipantelis** [comment]
Please correct me if I am wrong but the R tutorials are unreachable in the new website. All the links in http://mxnet.io/api/r/ lead to Python resources.

**Comment by piiswrong** [comment]
@indhub @domdivakaruni @madjam @bhavinthaker 
Let's put these tutorials back ASAP.

**Comment by hadjipantelis** [comment]
@piiswrong Any chance you fixed that broken link?

**Comment by thirdwing** [comment]
@hadjipantelis The R tutorials need some revision and we are working on it.

You can find some vignettes from https://github.com/dmlc/mxnet/tree/master/R-package/vignettes

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
Also, do please check out our [forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) for general "how-to" questions.

**Comment by edmBernard** [comment]
Is there a updated list of tutorial that still need improvement ?

**Comment by vishaalkapoor** [comment]
Hey @indhub, do you have info regarding @edmBernard's question? Thank you!

**Comment by anirudhacharya** [comment]
Can we update this issue with the current status or close it since we anyway will move to the new website pretty soon and we can track the tutorials there separately?

@indhub @szha @ThomasDelteil @vishaalkapoor

**Comment by anirudhacharya** [comment]
Most of the R tutorials in the list have been fixed. FYI

**Comment by bkmgit** [comment]
Is there still interest in the following tutorials:
* [How do I work with MXNet within a Matlab environment](https://github.com/apache/incubator-mxnet/tree/master/matlab)?
* [Can I run MXNet in a browser using JavaScript](https://github.com/dmlc/mxnet.js/)?
* [Neural Art - Transfer the style of one image onto the content the content of another image](https://github.com/apache/incubator-mxnet/tree/master/example/neural-style)

Might tutorials using [Octave](https://www.gnu.org/software/octave/) and [MSG-net](https://github.com/apache/incubator-mxnet/tree/master/example/gluon/style_transfer) be possible alternatives?

**Comment by ChaiBapchya** [comment]
I'm guessing the more tutorials we have (on varied topics) the better it is for our users. Personally, I'd interested in knowing all of these.
